domains/tan.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "tan.is-a.dev",
+  "owner": {
+    "username": "cuno92",
+    "email": "hoangvannhattan@gmail.com"
+  },
+  "record": {
+    "CNAME": "white-hill-030c00b00.4.azurestaticapps.net"
```

Comment:
can you show a preview of your website?
i only see a microsoft azure page that says "Your site will be ready soon"

Reply:
Dear @Stef-00012 , i added the preview of my website. Thanks!